frontend/src/views/organisation/workflows/MoOrganisationUnitTerminate.vue

```diff
@@ -101,7 +102,14 @@ export default {
 
     ...mapGetters({
       orgUnitDetails: STORE_KEY + '/GET_DETAILS'
-    })
+    }),
+
+    validity () {
+      return {
+        'from': this.terminate.validity.to,
```

Comment:
Add a comment explaining that this is an instantaneous validation, and hence the intentional duplicate `to`s?